### ISSUE TITLE
netutil/tls.go: Incorrect paths to tlsKeyFile and tlsCertFile do not cause a startup error

### ISSUE THREAD

**Issue by psmolkin** [VictoriaMetrics/VictoriaMetrics#6608]
### Describe the bug

Incorrect paths to `tlsKeyFile` and `tlsCertFile` do not cause a startup error.
Should be fixed here:
https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/lib/netutil/tls.go#L14

### To Reproduce

run VMAgent with extra args:
```bash
--tls=true --tlsKeyFile=/etc/RandomPath --tlsCertFile=whatever
```
Service successfully starts and listens the 8429 port, but connections to it fail with the non-obvious openssl errors

### Version

up to 1.102.0-rc2

**Comment by hagen1778** [comment]
@psmolkin this is expected behavior - see discussion here https://github.com/VictoriaMetrics/VictoriaMetrics/pull/6621 and specific comment here https://github.com/VictoriaMetrics/VictoriaMetrics/pull/6621#issuecomment-2225482188.

**Comment by psmolkin** [comment]
Not sure it's the same. This is a single shot func that is called during startup, so an erroneous configuration can't be fixed during runtime.

**Comment by hagen1778** [comment]
Files specified via `-tlsKeyFile` and `-tlsCertFile` cmd-line flags will be validated on the process start-up in https://github.com/VictoriaMetrics/VictoriaMetrics/commit/e5d279bb71e83e764eae13203d72693468537a67. This commit will be included into the next release.

**Comment by f41gh7** [comment]
this bug should be fixed in [v1.102.1](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.102.1).

feel free to reopen if you still observe issue
